x-pack/test/observability_ai_assistant_functional/tests/feature_controls/assistant_security.spec.ts

```diff
@@ -131,9 +132,16 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await testSubjects.missingOrFail(ui.pages.links.globalHeaderButton);
       });
       it('shows no AI Assistant conversations link in global search', async () => {
+        await PageObjects.common.navigateToUrl('home', '', {
```

Comment:
The test used to pass locally and fail on CI. With a suggestion from @flash1293, this test case doesn't depend on others)

Reply:
Should the navigation go in the `before` hook or will that cause problems for the other tests?